src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -4,6 +4,35 @@
 
 <entry_id>
 
+  <!-- NOTE that the description block determines what DATM% values can appear in the compset name 
```

Comment:
I am confused by this change - as far as I know there is no code in place that enforces the requirement that the description determines what values are allowed.  If you intend to introduce that in another PR then I think that these changes in the description in xml files should wait for that other PR.